lib/msal-browser/src/controllers/UnknownOperatingContextController.ts

```diff
@@ -0,0 +1,530 @@
+                `getAccountByLocalId: Returning signed-in accounts matching localAccountId: ${localAccountId}`
+            );
+            return account;
+        } else {
+            this.logger.verbose(
+                "getAccountByLocalId: No matching account found, returning null"
+            );
+            return null;
+        }
+    }
+    getAccountByUsername(username: string): AccountInfo | null {
+        this.logger.trace("getAccountByUsername called");
+        if (!username) {
+            this.logger.warning("getAccountByUsername: No username provided");
+            return null;
+        }
+
+        const account = this.browserStorage.getAccountInfoFilteredBy({
+            username,
+        });
+        if (account) {
+            this.logger.verbose(
+                "getAccountByUsername: Account matching username found, returning"
+            );
+            this.logger.verbosePii(
+                `getAccountByUsername: Returning signed-in accounts matching username: ${username}`
+            );
+            return account;
+        } else {
+            this.logger.verbose(
+                "getAccountByUsername: No matching account found, returning null"
+            );
+            return null;
+        }
+    }
+    getAllAccounts(): AccountInfo[] {
+        BrowserUtils.blockAPICallsBeforeInitialize(this.initialized);
+        return [];
+    }
+    initialize(): Promise<void> {
+        this.initialized = true;
+        return Promise.resolve();
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    acquireTokenPopup(request: PopupRequest): Promise<AuthenticationResult> {
+        BrowserUtils.blockAPICallsBeforeInitialize(this.initialized);
+        BrowserUtils.blockNonBrowserEnvironment(this.isBrowserEnvironment);
+        return {} as Promise<AuthenticationResult>;
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    acquireTokenRedirect(request: RedirectRequest): Promise<void> {
+        BrowserUtils.blockAPICallsBeforeInitialize(this.initialized);
+        BrowserUtils.blockNonBrowserEnvironment(this.isBrowserEnvironment);
+        return Promise.resolve();
+    }
+    acquireTokenSilent(
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        silentRequest: SilentRequest
+    ): Promise<AuthenticationResult> {
+        BrowserUtils.blockAPICallsBeforeInitialize(this.initialized);
+        BrowserUtils.blockNonBrowserEnvironment(this.isBrowserEnvironment);
+        return {} as Promise<AuthenticationResult>;
+    }
+    acquireTokenByCode(
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        request: AuthorizationCodeRequest
+    ): Promise<AuthenticationResult> {
+        BrowserUtils.blockAPICallsBeforeInitialize(this.initialized);
+        BrowserUtils.blockNonBrowserEnvironment(this.isBrowserEnvironment);
+        return {} as Promise<AuthenticationResult>;
+    }
+    acquireTokenNative(
```

Comment:
Shouldn't need this, not a public API

Reply:
Yep.  Left over from the stuff that leaked in previously.  I'll remove.